manageiq-operator/api/v1alpha1/helpers/miq-components/app-secret.go

```diff
@@ -37,8 +37,11 @@ func ManageAppSecret(cr *miqv1alpha1.ManageIQ, client client.Client, scheme *run
 }
 
 func defaultAppSecret(cr *miqv1alpha1.ManageIQ) *corev1.Secret {
+	encryptionKey := generateEncryptionKey()
+	token, _ := generateJwtTokenToken(encryptionKey)
 	secretData := map[string]string{
-		"encryption-key": generateEncryptionKey(),
+		"encryption-key": encryptionKey,
+		"token":          token,
```

Comment:
Wondering should this be something like "opentofu-runner-token" since "token" is very general

Reply:
Initially, I thought to name it opentofu-runner-token, but then I realized it is a JWT token that anybody can use. That's why I named it token. Although I couldn't think of any use case other than for OpenTofu-Runner, I can update it if necessary.

Reply:
Yeah that's true, maybe the name should be more general then? https://github.com/ManageIQ/manageiq-pods/pull/1115/files#diff-9abe24d14f8b59d578f4a765fdb3c0e6dfaadb6d3d383555e550b1a11d01ea7fR39

Reply:
Agree... I am changing it to `opentofu-runner-token. There is no other use case at this moment.

Reply:
Changing it to `terraform-runner-token`. We are using it by that name in ManageIQ